src/lib.rs

```diff
@@ -83,26 +89,27 @@ pub fn parse_identifier(input: ParseInput) -> ParseResult<Identifier> {
     spanned(p).parse(input)
 }
 
-/// Parses a single param, e.g., `x: u32`.
-fn parse_param(input: ParseInput) -> ParseResult<Parameter> {
+/// Parses a variable declaration, e.g., `x : u32`.
```

Comment:
Written DSLX is more styled as `x: u32`...but if `x : u32` is more Rust style, then we should head in that direction, I guess.

Reply:
Changed.

No, rust format does: `parse_variable_declaration(input: ParseInput)`

This is just me writing an example that parses. I don't think tiny examples in the documentation of our parser need to be particularly DSLX idiomatic.